### ISSUE TITLE
[1.0.2] Undefined array key "rut"

### ISSUE THREAD

**Issue by noquierouser** [Laragear/Rut#5]
### PHP & Platform

8.1.3

### Laravel verion

9.3.0

### Have you done this?

- [X] I am willing to share my stack trace and logs
- [X] I can reproduce this bug in isolation (vanilla Laravel install)
- [ ] I can suggest a workaround as a Pull Request

### Expectation

Livewire validation passes (or fails) the `rut` property/cast if a model uses the `HasRut` trait.

### Description

Laravel/Livewire returns a `ErrorException` indicating `Undefined array key "rut"` when attempting [real-time validation](https://laravel-livewire.com/docs/2.x/input-validation#real-time-validation) (`$this->validateOnly()`), and clears the `<rut>` input field when the `$this->validate()` method is executed on form submit.

### Reproduction

```php
// https://github.com/noquierouser/laragear-rut-livewire-bug
```

### Are you a Patreon supporter?

No, don't give priority to this

**Comment by DarkGhostHunter** [comment]
It appears that Livewire retrieves a 'rut_num' and 'rut_vd' keys instead of 'rut' keys. I assume this is retrieved from the model itself.

There is a chance there is nothing I can do and the bug should be remitted to Livewire repo instead.

I'm still investigating.

**Comment by DarkGhostHunter** [comment]
Okay, the cause of this is because, when Livewire retrieves the list of attributes from the Model to infuse with the validated data, it believes the validated `rut` attribute is on the Model `rut` attributes. This is not the case.

This is easily fixed using `$appends = ['rut']` to the model.

I'm adding two fixes. One for the validation rule to use `parse` instead of check, and another to automatically append the `rut` and hide the `rut_num` and `rut_vd` from serialization.

**Comment by DarkGhostHunter** [comment]
[Fixed in 1.1.0](https://github.com/Laragear/Rut/releases/tag/v1.1.0).

Ensure you're [appending the `rut`](https://github.com/Laragear/Rut#appending-a-rut).
